cmd/pd-server/main.go

```diff
@@ -254,7 +254,7 @@ func start(cmd *cobra.Command, args []string, services ...string) {
 
 	err = join.PrepareJoinCluster(cfg)
 	if err != nil {
-		log.Fatal("join meet error", errs.ZapError(err))
+		log.Panic("join meet error", errs.ZapError(err))
```

Comment:
I have some questions:
1. The behavior is different from the previous 
2. Will it be printed out if we set the log level to fatal?
3. What if the panic has a recover hook?

Reply:
1. Yes
2. Yes, the log will be printed when log level is fatal
3. If exist a recover hook, the panic will be caught.

> from: https://www.quasilyte.dev/blog/post/log-fatal-vs-log-panic/
> The log.Panic vs log.Fatal is essentially panic vs os.Exit(1). The latter will not let deferred calls to run.